pwndbg/heap/ptmalloc.py

```diff
@@ -1031,7 +1031,7 @@ def main_arena(self):
                             tmp_arena = self.malloc_state(tmp_next)
                             try:
                                 tmp_next = int(tmp_arena["next"])
-                            except gdb.MemoryError:
```

Comment:
Any chance we can elaborate more why we do catch all of those exceptions? Can we add comments that will mention when they occur?

Also any chance we can have tests that trigger those behaviors? 😅

Reply:
I will merge this for now, but pls consider sending another PR with additional comments or/and tests :P

Reply:
> Can we add comments that will mention when they occur?

Ahh, I'm sorry about that, I'll open another PR with additional comments about this.